src/components/common/RegPeopleAndRoles.vue

```diff
@@ -18,36 +18,41 @@
       <div class="subhead">Your application must include the following:</div>
       <ul>
         <template v-for="(rule, index) in getPeopleAndRolesResource.rules">
-          <li v-if="rule.id === RuleIds.NUM_COMPLETING_PARTY" :key="index">
-            <v-icon v-if="validNumCompletingParty" color="green darken-2" class="cp-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="cp-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
-          <li v-if="rule.id === RuleIds.NUM_PROPRIETORS" :key="index">
-            <v-icon v-if="validNumProprietors" color="green darken-2" class="proprietor-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="proprietor-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
-          <li v-if="rule.id === RuleIds.NUM_PARTNERS" :key="index">
-            <v-icon v-if="validNumPartners" color="green darken-2" class="partner-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="partner-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
-          <li v-if="rule.id === RuleIds.NUM_APPLICANT_PERSON" :key="index">
-            <v-icon v-if="validApplicantPerson" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="dir-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
-          <li v-if="rule.id === RuleIds.NUM_APPLICANT_ORG" :key="index">
-            <v-icon v-if="validApplicantOrg" color="green darken-2" class="dir-valid">mdi-check</v-icon>
-            <v-icon v-else-if="getShowErrors" color="error" class="dir-invalid">mdi-close</v-icon>
-            <v-icon v-else>mdi-circle-small</v-icon>
-            <span class="rule-item-txt">{{rule.text}}</span>
-          </li>
+          <RuleListItem class="completing-party-rule"
+            v-if="rule.id === RuleIds.NUM_COMPLETING_PARTY"
+            :key="index"
+            :valid="validNumCompletingParty"
+            :showErrors="getShowErrors"
+            :text="rule.text"
+          />
```

Comment:
These sub-components might not save much code vertical space, but they're a lot simpler.

Reply:
That's definitely cleaner!